### PR TITLE
[docs]-update-pre-provisioned-commands

### PR DESCRIPTION
**Pull request by gflorinskaya**
http://docs-d2iq-com-pr-4167.s3-website-us-west-2.amazonaws.com/

### CODE REVIEW

File: pages/dkp/konvoy/2.1/choose-infrastructure/pre-provisioned/define-infrastructure/index.md [hunk 1]
```diff
@@ -105,23 +83,30 @@ hunter-aws-cluster-pf4a3
...
 1.  Apply the new infrastructure file with the command:
 
-    ```shell
+    ```bash
     envsubst < preprovisioned_inventory.yaml | kubectl apply -f -
     ```
+    ```sh
+    preprovisionedinventory.infrastructure.cluster.konvoy.d2iq.io/my-preprovisioned-cluster-control-plane created
+    preprovisionedinventory.infrastructure.cluster.konvoy.d2iq.io/my-preprovisioned-cluster-md-0 created
+    ```
```
Comment (dlipovetsky) [inline pages/dkp/konvoy/2.1/choose-infrastructure/pre-provisioned/define-infrastructure/index.md:92]: Thanks for adding the output!

**Comment by dlipovetsky** [comment]
I propose removing these points from the "prerequisites" page, because they don't apply to the machine where the user runs `dkp`, and the points are covered in detail the "define infrastructure" page. WDYT?

```
* Stop and disable firewalld systemd service.
* Turn off memory swap.
```

**Review by dlipovetsky — APPROVED** [review]

**Review by cneth — APPROVED** [review]

**Comment by gflorinskaya** [comment]
> I propose removing these points from the "prerequisites" page, because they don't apply to the machine where the user runs `dkp`, and the points are covered in detail the "define infrastructure" page. WDYT?
> 
> ```
> * Stop and disable firewalld systemd service.
> * Turn off memory swap.
> ```

I agree. This is not really a prerequisite if we describe how to do it in the middle of the procedure.

**Comment by msdolbey** [comment]
> > I propose removing these points from the "prerequisites" page, because they don't apply to the machine where the user runs `dkp`, and the points are covered in detail the "define infrastructure" page. WDYT?
> > ```
> > * Stop and disable firewalld systemd service.
> > * Turn off memory swap.
> > ```
> 
> I agree. This is not really a prerequisite if we describe how to do it in the middle of the procedure.

@gflorinskaya @dlipovetsky - I don't see a note about turning off memory swap - I'll add a line.

**Comment by msdolbey** [comment]
okay - cool - I just added your changes @gflorinskaya to the 2.2 version - I think we're cooking with gas now.

**Review by msdolbey — APPROVED** [review]
👍 
all the provisions

**Pull request merged**